python-packages/sra_client/CHANGELOG.json

```diff
@@ -1,11 +1,22 @@
 [
     {
-        "timestamp": 1553491629,
         "version": "1.0.1",
         "changes": [
             {
-                "note": "Fix regex validation on numeric values"
+                "note": "Fix regex validation on numeric values",
+                "pr": 1731
             }
-        ]
+        ],
+        "timestamp": 1553491629
+    },
+    {
+        "version": "1.0.2",
```

Comment:
You can combine all these changes into one version bump, since we haven't published 1.0.1 yet anyways.

Reply:
https://github.com/0xProject/0x-monorepo/pull/1734/commits/4883dda8e02bd26975e25859231c1a7e1b4200a8